test/restrictingGuards.spec.ts

```diff
@@ -109,6 +180,29 @@ describe('restricting guards', function() {
 		it('guards against numbers outside the range', function() {
 			assertGuards(false)(guard, -1);
 			assertGuards(false)(guard, 1);
+			assertGuards(false)(guard, Number.NEGATIVE_INFINITY);
+			assertGuards(false)(guard, Number.POSITIVE_INFINITY);
+			assertGuards(false)(guard, Number.NaN);
+		});
+	});
+	context('safe', function() {
+		const guard = restricting.numberIsSafeInteger;
+		it('guards for integers in the range', function() {
+			assertGuards(true)(guard, Number.MIN_SAFE_INTEGER);
+			assertGuards(true)(guard, Number.MAX_SAFE_INTEGER);
+			assertGuards(true)(guard, 0);
+		});
+		it('guards for integers outside the range', function() {
```

Comment:
We typically phrase this as guards *against*.

Reply:
Yep, I caught that on other ones but forgot I'd done this one with the wrong phrasing, fixed ✔️